lawnchair/src/app/lawnchair/ui/preferences/PreferencesDashboard.kt

```diff
@@ -40,6 +44,7 @@ fun PreferencesDashboard() {
         backArrowVisible = false,
         actions = { PreferencesOverflowMenu() }
     ) {
+        PreferencesDebugWarning()
```

Comment:
Move the `BuildConfig.DEBUG` check here instead so it gets eliminated entirely in release builds.